develop-docs/integrations/msteams/index.mdx

```diff
@@ -14,8 +14,8 @@ You will need to be a Microsoft Teams Administrator to perform some of the requi
 
 1. Visit the [Developer Portal](https://dev.teams.microsoft.com/apps). Note: You may have to sign up for the [MS 365 Developer Program](https://developer.microsoft.com/en-us/microsoft-365/dev-program) if you're having trouble accessing this.
 2. Click "New App".
-3. Enter "Sentry" as the name.
-4. Under "Basic Information" use the following values (the rest may be blank):
+3. Name your app anything you like (e.g. `Sentry`).
```

Comment:
can u change this to make the name specific to them (so its easier to find your app)

Reply:
should i remove the example then?